menu/lib/twig-compiler.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
this file is not needed. should use the pattern-importer version directly from pattern-imnporter